packages/gossiplog/src/BranchMergeIndex.ts

```diff
@@ -20,12 +20,12 @@ export class BranchMergeIndex {
 			target_message_id: "string",
 			target_clock: "integer",
 		},
-	} satisfies ModelsInit
+	} satisfies ModelSchema
 
 	constructor(private readonly db: AbstractModelDB) {}
 
-	public async insertBranchMerge(entry: BranchMergeEntry) {
-		const id = `${entry.source_branch}:${entry.source_message_id}:${entry.target_branch}:${entry.target_message_id}`
-		await this.db.set("$branch_merges", { id, ...entry })
+	public async insertBranchMerge(record: BranchMergeRecord) {
+		const id = `${record.source_branch}:${record.source_message_id}:${record.target_branch}:${record.target_message_id}`
```

Comment:
since (branch, clock) uniquely identifies a message, can you use `${source_branch}:${source_clock}-${target_branch}:${target_clock}` for the merge record IDs? there will be lots of these entries so eliminating 64 characters per ID would be nice

Reply:
yes, this would work too!